test/integration/auth/node_test.go

```diff
@@ -249,6 +252,18 @@ func TestNodeAuthorizer(t *testing.T) {
 		})
 	}
 
+	updatePVCCapacity := func(client clientset.Interface) error {
+		patchBytes := []byte(`{"status":{"capacity": {"storage": "500Gi"}}}`)
```

Comment:
make the storage increment on each call so the patch always modifies the object

```
storage := 500
updatePVCCapacity := func(client clientset.Interface) error {
  storage += 1
  patchBytes := []byte(`{"status":{"capacity": {"storage": fmt.Sprintf("%dGi",storage)}}}`)
```

Reply:
fixed